README.md

```diff
@@ -33,23 +33,31 @@
 
  2. Build and install librats
 
+ ```shell
+ cd rats-tls
+ cmake -DBUILD_SAMPLES=on -H. -Bbuild
+ make -C build install
+ ```
+
+ `cert-app` will be installed to `/usr/share/librats/samples/cert-app` on your system. All instances are placed in `/opt/rats-tls/lib`.
```

Comment:
please change `rats-tls` to `librats` .  I remember the path is `/usr/local/lib/librats/`, please check it. Thank you